docs/operators/transform.md

```diff
@@ -0,0 +1,121 @@
+Transform - Operator Documentation
+==================================
+
+About Transform operator
+------------------------
+
+Transform means mapping of field expression from input to output or conversion of fields from one type to another.
+This operator is stateless. This operator can transform input POJO using provided expression mappings into output POJO.
+
+Use Case
+--------
+
+Consider the data received from input source and want to transform the data as below output schema while emitting to downstream operator. 
+
+Consider the below input schema
+`{FirstName, LastName, Phone, DateOfBirth, Address}`
+
+and output schema
+`{Name, Phone, Age, Address}`
+
+Suppose the requirement as follows:
+    -   "Name" field would be the combination of FirstName and LastName 
+    -   "Age" can be calculated from DateOfBirth.
+            
+Then, below are the expression mappings between output and input fields. 
+
```

Comment:
Suggest rewriting the above text to avoid heavily overloaded words like "schema" and explain what is going on in simple terms, for example:

Consider input objects with these fields:
`{FirstName, LastName, Phone, DateOfBirth, Address}` and output objects
with fields: `{Name, Phone, Age, Address}`. Suppose `Name` is a concatenation of `FirstName`
and `LastName` and `Age` is computed by subtracting the `DateOfBirth` from the current year.

These simple computations can be expressed as Java expressions where the input object is
represented by `$` and provided as configuration parameters as follows: